EBMOL/hydro_ebmol.cpp

```diff
@@ -244,7 +244,7 @@ EBMOL::ComputeAofs ( MultiFab& aofs, int aofs_comp, int ncomp,
                 // If convetive form is required, the next parallel for
                 // will take care of it.
                 Real mult = 1.0;
-                std::vector<int>  div_iconserv(ncomp,1);
+                Gpu::DeviceVector<int>  div_iconserv(ncomp,1);
```

Comment:
@mic84 Why does EBMOL pass all conservative here in ComputeAofs(), when all the others (MOL, Godunov, EBGodunov) do not. At a minimum, EBMOL should be consistent with MOL here (since this is for the EB regular case).

Reply:
@cgilet For EBMOL to support convective differencing, we do `u*grad(S) = div(uS) - S*div(u)`, where `div(u)` is user provided. Therefore the conservative form of the divergence is computed first and then, if the convective form is required, the extra term `-S*div(u)` is added to the `Aofs` (see comment lines in the above snippet). I agree that we should use the same approach with plain MOL, and probably even with Godunov.